plugins/backend/catalog-backend-module-aws/src/index.ts

```diff
@@ -16,3 +16,5 @@
 
 export * from './providers';
 export * from './processors';
+export * from './types';
+export * from './utils/templating';
```

Comment:
![Passing: wanted](https://gh-labels.dancrews.com/passing/wanted.svg)
During implementation on my own, these were things I sure wished to have had easy access to but didn't.